framework/src/application/application.ts

```diff
@@ -317,8 +317,6 @@ export class Application {
 		} catch (error) {
 			this.logger.fatal({ err: error as Error }, 'failed to shutdown');
 		}
-
-		process.exit(errorCode);
```

Comment:
We had handled the error up and we are not throwing it upward, and even we removed this `process.exit` here. How the main process will exit in case of error?

Reply:
if all the open handle was properly closed, it should be able to exit the process, but it was not the case. I will revert the the change here

Reply:
I think open handles also count for the files. And we have logging file open always, that may be the cause of it.